pages/_app.tsx

```diff
@@ -11,16 +11,16 @@ Sentry.init({ dsn: 'https://37c553e6cb9c4d7b91443519ccd448a9@sentry.io/1514998'
 initializeGoogleAnalytics();
 
 export default class MyApp extends App {
-  componentDidCatch(error, errorInfo) {
+  componentDidCatch(error: Error, errorInfo: React.ErrorInfo) {
     Sentry.withScope((scope) => {
-      Object.keys(errorInfo).forEach((key) => {
+      (Object.keys(errorInfo) as (keyof typeof errorInfo)[]).forEach((key) => {
         scope.setExtra(key, errorInfo[key]);
       });
       if (process.env.NODE_ENV === 'production') {
         Sentry.captureException(error);
       }
     });
-    super.componentDidCatch(error, errorInfo);
+    super.componentDidCatch && super.componentDidCatch(error, errorInfo);
```

Comment:
can we do? `super?.componentDidCatch(error, errorInfo)`

Reply:
I've converted it to this `super.componentDidCatch?.(error, errorInfo)`